tests/related.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { configure, shallow } from 'enzyme';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
+configure({ adapter: new Adapter() });
+
+import RelatedProductsList from '../src/components/Related/RelatedProductsList';
+
+
+describe("Related Products List", () => {
+  let wrapper, wrapperState, wrapperInstance;
+
+  beforeEach(() => {
+    wrapper = shallow(<RelatedProductsList />);
+    wrapperState = wrapper.state();
+  });
+
+  it('is a stateful component', () => {
```

Comment:
I didn't know `it` was valid jest syntax?

Reply:
It is- essentially an alias for 'test'. I understand the intention is for the test to read as closely to plain english as possible, so considering the format of test names, I chose to go with "it" instead of "test"